libcontainer/configs/config_test.go

```diff
@@ -170,46 +200,83 @@ func TestCommandHookRun(t *testing.T) {
 		t.Fatal(errJ)
 	}
 
-	verifyCommandTemplate := `
-	if [ $0 != cmdname ]; then
-		echo "Bad value for \$0. Expected 'cmdname', found '$0'"
-		exit 1
-	fi
-
-	if [ $1 != testarg ]; then
-		echo "Bad value for \$1. Expected 'testarg', found '$1'"
-		exit 1
-	fi
-
-	if [ -z $FOO ] || [ $FOO != BAR ]; then
-		echo "Bad value for FOO. Expected 'BAR', found '$FOO'"
-		exit 1
-	fi
-
-	expectedJson=%q
-
-	read JSON
-	if [ $JSON != $expectedJson ]; then
-		echo "Bad JSON received. Expected '$expectedJson', found '$JSON'"
-		exit 1
-	fi
-
-	exit 0
+	verifyCommandTemplate := `#!/bin/sh
+if [ $1 != "testarg" ]; then
```

Comment:
```suggestion
if [ "$1" != "testarg" ]; then
```

Reply:
I added all of them manually in a single commit.